src/api/app/jobs/project/update_packages_if_dirty_job.rb

```diff
@@ -0,0 +1,9 @@
+class Project
+  class UpdatePackagesIfDirtyJob < ApplicationJob
+    self.priority = 10
```

Comment:
It seems that priority is used to determine the priority between jobs within a queue. So the default priority is 0, so if there is a queue with priority 0 and another job with priority 10, then the 0 priority job gets completed first.

https://github.com/collectiveidea/delayed_job#parameters